src/components/BorrowerProfile/ShareButton.vue

```diff
@@ -8,6 +8,7 @@
 		:utm-content="utmContent"
 		:linked-in-title="linkedInTitle"
 		:open-lightbox="forceLightbox"
+		:loan-id="loan.id"
```

Comment:
Do we need to add this to the `ThanksPageEcoChallenge` as well? Not sure if that page is still active.

Reply:
Good call, but no. That challenge is no longer active, and the share on that page shares a climate fact, not a loan.